Lib/multiprocessing/connection.py

```diff
@@ -181,6 +181,13 @@ def close(self):
             finally:
                 self._handle = None
 
+    def _detach(self):
+        """Stop managing the underlying file descriptor or handle."""
+        try:
+            return self._handle
```

Comment:
We don't ever use the return value in this PR.  Is there a plan to use it elsewhere?

Reply:
no plan, return removed.